govuk_chat_evaluation/rag_answers/custom_deepeval/metrics/factual_correctness_completeness/factual_correctness_completeness.py

```diff
@@ -19,7 +20,12 @@
 import logging
 
 
-class FactualCorrectnessMetric(BaseMetric):
+class Mode(Enum):
+    CORRECTNESS = auto()
+    COMPLETENESS = auto()
```

Comment:
This `auto()` assigns a integer value (1 and 2). I think it is ok here as the enumerations are only used for branching logic in the metric's code, so the enum’s actual underlying value is irrelevant. And if we had to later rename the members, nothing will break.

And I do not think we log this or nothing where a human needs to read them, so all's good.